src/windows/models/emoji_model.py

```diff
@@ -153,68 +158,45 @@ def update_model(self, clear=True):
 
                 row = []
 
-                # Append thumbnail
-                col = QStandardItem()
-                col.setIcon(QIcon(thumb_path))
-                col.setText("%s (%s)" % (emoji_name, emoji_type))
-                col.setToolTip(emoji_name)
-                col.setData(emoji_type)
-                col.setFlags(Qt.ItemIsSelectable | Qt.ItemIsEnabled | Qt.ItemIsUserCheckable | Qt.ItemIsDragEnabled)
-                row.append(col)
-
-                # Append Filename
+                # Set emoji data
                 col = QStandardItem("Name")
-                col.setData(emoji_name, Qt.DisplayRole)
+                col.setIcon(QIcon(thumb_path))
                 col.setText(emoji_name)
+                col.setToolTip(emoji_name)
+                col.setData(path)
                 col.setFlags(Qt.ItemIsSelectable | Qt.ItemIsEnabled | Qt.ItemIsUserCheckable | Qt.ItemIsDragEnabled)
                 row.append(col)
 
-                # Append Media Type
-                col = QStandardItem("Type")
-                col.setData(type, Qt.DisplayRole)
-                col.setText(emoji_type)
-                col.setFlags(Qt.ItemIsSelectable | Qt.ItemIsEnabled | Qt.ItemIsUserCheckable | Qt.ItemIsDragEnabled)
-                row.append(col)
-
-                # Append Path
-                col = QStandardItem("Path")
-                col.setData(path, Qt.DisplayRole)
-                col.setText(path)
-                col.setFlags(Qt.ItemIsSelectable | Qt.ItemIsEnabled | Qt.ItemIsUserCheckable | Qt.ItemIsDragEnabled)
+                # Append filterable group
+                col = QStandardItem(emoji_type)
                 row.append(col)
```

Comment:
The only other column I create is one that contains the `emoji_type` (group), because I used that for filtering with the dropdown selector. I'll go into that in more detail in a bit, too.